app-text/scdoc/scdoc-1.2.3.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit toolchain-funcs
+
+DESCRIPTION="Standalone tool for generating man pages with a simple syntax"
+HOMEPAGE="https://git.sr.ht/~sircmpwn/scdoc"
+SRC_URI="https://git.sr.ht/~sircmpwn/scdoc/snapshot/${P}.tar.xz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+src_prepare() {
+	sed -i 's/-Werror//' Makefile || die 'Failed to remove -Werror'
+
+	sed -i 's/CFLAGS=/CFLAGS+=/' Makefile || die 'Failed to patch makefile'
+
+	default
+}
+
+src_compile() {
+	if tc-is-cross-compiler; then
+		tc-export_build_env
+		MAKEOPTS+=" HOST_SCDOC=./hostscdoc"
```

Comment:
mucking with `MAKEOPTS` isn't allowed (I know this is common, still not allowed), append it to the `emake` line